.github/workflows/ci.yml

```diff
@@ -32,8 +32,6 @@ jobs:
   upload-sha:
     name: Upload sha commit tag to GCS
     needs: [build]
-    # Make uploads available only from main (needed for deployments)
```

Comment:
We don't want to automatically deploy every push to each branch. To upload a PR add "upload" label to you PR.